api/src/test/scala/dcos/metronome/api/WithApplicationComponents.scala

```diff
@@ -121,6 +122,8 @@ class MockApiComponents(context: Context) extends BuiltInComponentsFromContext(c
 
   lazy val assets: Assets = wire[Assets]
 
+  lazy val queueService: LaunchQueueService = QueueServiceFixture.simpleQueueService()
```

Comment:
I have removed this locally and code still compiles. Since it is not used anywhere, I would drop it...